.gitignore

```diff
@@ -0,0 +1,3 @@
+test.md
+
+//this file should be ignored by git
```

Comment:
Great job implementing most of the requirements nicely. 

However, I see that you do not have the `node_modules` folder added to `.gitignore` file. 
Perhaps, it is because you have not started installing dependencies yet. But it is best practice to add it on the get-go to avoid situations where you commit your work alongside the `node_module` folder.

- [ ] Kindly add `node_modules` folder to the `.gitignore` file.

[See why you should .gitignore node_modules folder](https://dev.to/grawl/why-you-dont-have-to-commit-nodemodules-folder-33nm)